source/Engine.cpp

```diff
@@ -447,7 +447,7 @@ void Engine::Step(bool isActive)
 	else if(flagship)
 	{
 		center = flagship->Position();
-		centerVelocity = flagship->Velocity();
+		centerVelocity = flagship->IsDestroyed() ? Point(0,0) : flagship->Velocity();
```

Comment:
The default value of a point is already (0,0). (See a few lines above, `centerVelocity = Point();`.)
```suggestion
		centerVelocity = flagship->IsDestroyed() ? Point() : flagship->Velocity();
```

I think it might make more sense though to set the velocity of a destroyed ship to 0, instead of always checking if the flagship is destroyed. That is, add `velocity = Point();` here, say after L1294.
https://github.com/endless-sky/endless-sky/blob/29caeb623f574eeab2ad1e46bb654e6749c7f623/source/Ship.cpp#L1291-L1296

Reply:
Makes sense, moved to ship:: so it dosnt need to check every engine::step() tested and has same end effect.